src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -205,8 +205,11 @@ public void initialiseTags() {
         for (Person person : persons) {
             Set<Tag> tagForPerson = person.getTags();
             for (Tag tag : tagForPerson) {
+                tag.increaseTaggedCount();
```

Comment:
I think can also add this below for the `initialiseWeddings()` method to do `wedding.increaseWeddingCount()`